cirq-core/cirq/ops/gateset.py

```diff
@@ -406,9 +406,10 @@ def _value_equality_values_(self) -> Any:
 
     def __repr__(self) -> str:
         name_str = f'name = "{self.name}", ' if self.name is not None else ''
+        gates_str = f'{self._gates_repr_str}, ' if len(self._gates_repr_str) > 0 else ''
         return (
             f'cirq.Gateset('
-            f'{self._gates_repr_str}, '
+            f'{gates_str}'
```

Comment:
Why change here ?

Reply:
Please add a test to `gateset_test.py`. Also, If the change is unrelated and is not blocking this PR, then I'd prefer if we can split it out in a separate PR.

Reply:
Added and moved to https://github.com/quantumlib/Cirq/pull/5322